### DISCUSSION TITLE
Goals for v1.14.0

### DISCUSSION THREAD

**Discussion by WorldLanguages** [ScratchAddons/ScratchAddons#2202] [Discussion]
Listing what we should focus for next release:
1. Redesigning the settings page, try to reimagine how users should be able to configure addons, explore new ones, etc. Both in the full screen settings page and in the popup. One of the main improvements we should do IMO is provide a way to list addons as a grid and show screenshots on what it does, similarly to [scratchaddons.com/addons](https://scratchaddons.com/addons). Related issues: #475, #1663, #2031
2. Make more addons support `dynamicDisable` and `dynamicEnable`, which makes it easier for users to test addons without needing to reload each time. If we migrate most addons, we should add a modal that reminds the user to refresh if they enabled an addon that still requires a reload for technical reasons, like [turbowarp.org/addons](https://turbowarp.org/addons) does. Related issue: #1673
3. The haunted color picker. See the [milestone](https://github.com/ScratchAddons/ScratchAddons/milestone/19)
4. Better page for docs? Use a repo instead of the wiki tab

**Comment by mxmou** [comment]
> One of the main improvements we should do IMO is provide a way to list addons as a grid and show screenshots on what it does, similarly to [scratchaddons.com/addons](https://scratchaddons.com/addons).

This would be good, but not as a grid - I think a grid works well on a showcase page but it isn't practical to find things in one. It's better to make the screenshots smaller.

**Reply by WorldLanguages** [reply]
Yeah, another thing I forgot to write down is that now we have so many addons, _exploring addons_ and _configuring already enabled addons_ are separate actions, and we should keep that in mind. Related to #1505

**Comment by Secret-chest** [comment]
Then maybe a setup screen for first use?

**Reply by cobaltt7** [reply]
on first use the user is directed to https://scratchaddons.com/welcome already

**Reply by Secret-chest** [reply]
@RedGuy12 Yes but that page dosent have any addons

**Comment by TheColaber** [comment]
v2.0.0 👀

If anyone wants to help with `dynamicDisable` and `dynamicEnable`, greatly appreciated... We want to focus on recommended addons and addons that the user might enable/disable frequently.

**Comment by Secret-chest** [comment]
We cloud add images to addons.

**Comment by Secret-chest** [comment]
In the popup, we should be able to pin addons.

**Reply by mxmou** [reply]
See #1250 for that.

**Comment by JellyScript** [comment]
Is there a way to change the FPS?

**Reply by cobaltt7** [reply]
yes, the 60fps addon (it's any fps, not just 60)

**Comment by JellyScript** [comment]
Thank you! But, is there a way to do it without addons? I know you type
?fps=__ but it won't work.

**Reply by WorldLanguages** [reply]
That only works on turbowarp

**Comment by FunctionalMetatable** [comment]
> 4\. Better page for docs? Use a repo instead of the wiki tab

[material for mkdocs](https://github.com/squidfunk/mkdocs-material) would be nice as a page

**Comment by JellyScript** [comment]
Thank you, one final inquiry... When I open the HTML, I copy and paste it
directly?
